Community/Tdarr_Plugin_TD01_TOAD_Autoscan.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable linebreak-style */
+module.exports.dependencies = [
+  'request',
+];
+
+/* eslint no-plusplus: ["error", { "allowForLoopAfterthoughts": true }] */
+module.exports.details = function details() {
+  return {
+    id: 'Tdarr_Plugin_TD01_TOAD_Autoscan',
+    Stage: 'Post-processing',
+    Name: 'Trigger Plex_Autoscan.',
+    Type: 'Video',
+    Operation: '',
+    Description: `Connects to plex_autoscan and triggers a manual search within the files directory.\n\n
+    Works with the hotio autoscan docker that can be found here https://hotio.dev/containers/autoscan/ which \n\n
+    is based on https://github.com/cloudbox/autoscan`,
+    Version: '1.0',
+    Link: '',
+    Tags: '3rd party,post-processing,configurable',
+
+    Inputs: [{
+      name: 'autoscan_address',
+      tooltip: `
+               Enter the IP address/URL for autoscan. Must include http(s)://
+
+               \\nExample:\\n
+               http://192.168.0.10
+
+               \\nExample:\\n
+               https://subdomain.domain.tld`,
+    },
+    {
+      name: 'autoscan_port',
+      tooltip: `
+               Enter the port Autoscan is using, default is 3468
+
+               \\nExample:\\n
+               3468`,
+    },
+    {
+      name: 'autoscan_username',
+      tooltip: `
+               If authentication is configured, specify the username
+
+               \\nExample:\\n
+               Batman`,
+    },
+    {
+      name: 'autoscan_password',
+      tooltip: `
+              If authentication is configured, specify the password
+
+               \\nExample:\\n
+               SecretPassword`,
+    },
+    ],
+  };
+};
+
+module.exports.plugin = function plugin(file, librarySettings, inputs) {
+  // eslint-disable-next-line global-require,import/no-unresolved,import/no-extraneous-dependencies
+  const request = require('request');
+  // Set up required variables.
+  const ADDRESS = inputs.autoscan_address;
+  const PORT = inputs.autoscan_port;
+
+  if (inputs.autoscan_username) {
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Could you please remove this? Just disabling eslint isn't the way to solve this.